week-5/F-webchat/exercise-1.js

```diff
@@ -33,5 +33,25 @@ When you open index.html in your browser, it should display the existing message
 
 */
 
-
-// Write your code here
\ No newline at end of file
+// let myButton = document.querySelector("#submit");
+// myButton.addEventListener("click", sumbit);
+
+// function sumbit() {
+//   let reply =
+// }
+
+// Write your code here
+function displayMessage() {
+  fetch("https://codeyourfuture.herokuapp.com/api/messages")
+    .then(responds => {
+      return responds.text();
+    })
+    .then(data => {
+      document.querySelector("#message-list").innerHTML = data;
```

Comment:
Data is a very large string of many many messages. This is best handle one at a time.
`response.json().forEach(message => ...`